app/src/main/java/org/mozilla/fenix/settings/creditcards/controller/CreditCardsManagementController.kt

```diff
@@ -21,6 +21,8 @@ interface CreditCardsManagementController {
      * @see [CreditCardsManagementInteractor.onSelectCreditCard]
      */
     fun handleCreditCardClicked(creditCard: CreditCard)
+
+    fun handleAddCreditCardButtonClicked()
```

Comment:
Let's add the kDoc here

```suggestion
    /**
     * @see [CreditCardsManagementInteractor.onAddCreditCard]
     */
    fun handleAddCreditCardButtonClicked()
```